jbpm-workitems/jbpm-workitems-webservice/src/main/java/org/jbpm/process/workitem/webservice/WebServiceWorkItemHandler.java

```diff
@@ -533,11 +543,86 @@ public void abortWorkItem(WorkItem workItem,
     }
 
     private ClassLoader getInternalClassLoader() {
-        if (this.classLoader != null) {
-            return this.classLoader;
+        ClassLoader cl = this.classLoader != null ? classLoader : Thread.currentThread().getContextClassLoader(), parent = cl;
+        Collection<File> uris = new HashSet<>();
+        do {
+            if (parent.getClass().getSimpleName().equals("ModuleClassLoader")) {
+                try {
+                    getJarsFromModuleClassLoader(parent, uris);
+                } catch (ReflectiveOperationException e) {
+                    throw new IllegalStateException("Problem calculating classpath", e);
+                }
+            }
+            parent = parent.getParent();
+        } while (parent != null);
+        if (!uris.isEmpty()) {
+            cl = new CustomClassLoader(uris, cl);
+        }
+        return cl;
+    }
+
+    private static class CustomClassLoader extends URLClassLoader {
```

Comment:
says very little. You are bypassing the class loader problem of CXF. what about
CXFJavaCompilerClassLoader or so ?

Reply:
Thanks for the suggestion, I was unable to find a proper name ;)